### DISCUSSION TITLE
Welcome to Aista Magic Cloud Discussions!

### DISCUSSION THREAD

**Discussion by polterguy** [polterguy/magic#182] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RichToomey** [comment]
Interested in code gen from sql tools.

**Reply by polterguy** [reply]
Great, you've come to the right place :)

We're currently working on a visual DDL database designer tool for the record. It's not done yet, but will be released soon :)
